_data/simple-icons.json

```diff
@@ -6976,7 +6976,7 @@
         },
         {
             "title": "Lemmy",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

Comment:
```suggestion
            "hex": "14854F",
```
Might be a better choice in this case. What do you think?

Reply:
Sure.